installer/Templates/Env.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+
+namespace Installer\Templates;
+
+
+use Template;
+
+class Env implements Template
+{
+
+	public function print(array $input = [])
+	{
+		echo '<form method="post" action="?step=env">
+        <textarea class="textarea" name="envConfig">'.$input['env'].'</textarea>
```

Comment:
It's not immediately obvious that this is an editable area. Perhaps there should be a one-line header telling the user to edit the file as appropriate?

Reply:
fixed.